src/components/v5/frame/ColonyHome/ColonyHome.tsx

```diff
@@ -32,6 +33,10 @@ const ColonyHome = () => {
   const isMobile = useMobile();
   const selectedDomain = useGetSelectedDomainFilter();
   const teamsBreadcrumbs = useCreateTeamBreadcrumbs();
+  const [selectedAction, setSelectedAction] = useState<string | undefined>(
+    undefined,
+  );
+  const { defaultValues } = useGetActionData(selectedAction || undefined);
```

Comment:
Do we need two such states, in both `ColonyHome` and `ActivityPage` or could it live in `ColonyActionsTable` instead?

Reply:
@jakubcolony I tried to add it to the`ColonyActionsTable` but there was a problem with rerenders